package.json

```diff
@@ -187,7 +187,8 @@
     },
     "resolutions": {
         "@nx/js@16.3.2": "patch:@nx/js@npm%3A16.3.2#./.yarn/patches/@nx-js-npm-16.3.2-a543eca792.patch",
-        "postcss": "^8.4.31"
+        "postcss": "^8.4.31",
+        "word-wrap": "^1.2.4"
```

Comment:
is this updated by newer versions of our dependencies?

Reply:
yea its just to apply a patch change to fix a hard dep: 
<img width="606" alt="image" src="https://github.com/DevCycleHQ/js-sdks/assets/1219069/0e1f058b-f6bd-468a-b6ac-0b0951031a67">